### PR TITLE
feat(Learner Backpack): allow global header to receive learner backpack + add 2 icons

### PR DESCRIPTION
**Pull request by saghdaey**
### Overview

NO TIME; HACK HACK HACK! 

(I need an alphabuild)

Also added backpack icon

### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.47-alpha.e631d2.0<br/>@codecademy/gamut-labs@13.2.10-alpha.e631d2.0<br/>@codecademy/styleguide@33.6.1-alpha.e631d2.0<br/></pre>

File: packages/gamut-labs/src/experimental/GlobalHeader/types.tsx [hunk 1]
```diff
@@ -6,6 +6,10 @@ type RenderNotifications = {
   desktop: () => ReactNode;
   mobile: () => ReactNode;
 };
+type RenderLearnerBackpack = {
+  desktop: () => ReactNode;
```
Comment (saghdaey) [inline packages/gamut-labs/src/experimental/GlobalHeader/types.tsx:10]: mobile support -->  future!

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.2.47-alpha.093c4a.0<br/>@codecademy/gamut-labs@13.2.10-alpha.093c4a.0<br/>@codecademy/styleguide@33.6.1-alpha.093c4a.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6065ee4b83abae008a7553b3--gamut-preview.netlify.app>https://6065ee4b83abae008a7553b3--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6065ee4b83abae008a7553b3>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.5.1-alpha.482970.0<br/>@codecademy/gamut-kit@0.2.47-alpha.482970.0<br/>@codecademy/gamut-labs@13.2.10-alpha.482970.0<br/>@codecademy/gamut@29.7.1-alpha.482970.0<br/>@codecademy/styleguide@33.6.1-alpha.482970.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60661f152d597e48c1067516--gamut-preview.netlify.app>https://60661f152d597e48c1067516--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60661f152d597e48c1067516>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.5.1-alpha.17fbef.0<br/>@codecademy/gamut-kit@0.2.47-alpha.17fbef.0<br/>@codecademy/gamut-labs@13.2.10-alpha.17fbef.0<br/>@codecademy/gamut@29.7.1-alpha.17fbef.0<br/>@codecademy/styleguide@33.6.1-alpha.17fbef.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://6067352ac54c8dd65fbd1787--gamut-preview.netlify.app>https://6067352ac54c8dd65fbd1787--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/6067352ac54c8dd65fbd1787>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@4.14.2-alpha.2b86fb.0<br/>@codecademy/gamut-kit@0.3.113-alpha.2b86fb.0<br/>@codecademy/gamut-labs@18.1.4-alpha.2b86fb.0<br/>@codecademy/gamut@33.0.1-alpha.2b86fb.0<br/>@codecademy/styleguide@40.0.1-alpha.2b86fb.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60ad0e7c22e78800bd31b557--gamut-preview.netlify.app>https://60ad0e7c22e78800bd31b557--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60ad0e7c22e78800bd31b557>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.7.4-alpha.63a375.0<br/>@codecademy/gamut-kit@0.4.70-alpha.63a375.0<br/>@codecademy/gamut-labs@19.16.8-alpha.63a375.0<br/>@codecademy/gamut@35.1.1-alpha.63a375.0<br/>@codecademy/styleguide@42.1.1-alpha.63a375.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60f731208211010acaba0918--gamut-preview.netlify.app>https://60f731208211010acaba0918--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60f731208211010acaba0918>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@5.7.4-alpha.0b72c4.0<br/>@codecademy/gamut-kit@0.5.2-alpha.0b72c4.0<br/>@codecademy/gamut-labs@19.16.14-alpha.0b72c4.0<br/>@codecademy/gamut@35.4.2-alpha.0b72c4.0<br/>@codecademy/styleguide@42.3.2-alpha.0b72c4.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://610019ce7696a62623574858--gamut-preview.netlify.app>https://610019ce7696a62623574858--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/610019ce7696a62623574858>Deploy Logs</a></p>

**Pull request closed without merging**